app/views/posts/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<h3>This is a list of posts for a given usser</h3>
+
+<% if @user %>
+  <h1>User: <%= @user.name %></h1>
+  <p>Number of Posts: <%= @user.posts_counter %></p>
```

Comment:
- Good job done so far, but you do not have an attribute called `posts_counter` in the user model, please fix this 🙏